common/model-views.cpp

```diff
@@ -240,7 +240,7 @@ namespace rs2
         return res;
     }
 
-    bool option_model::draw(std::string& error_message)
+    bool option_model::draw(std::string& error_message, notifications_model& model)
```

Comment:
Is it possible to add report level to allow filtering in the notification panel?

Reply:
It's a good idea, but requires some work. In theory we can push much more information to this panel and add filtering options (like search and level based filtering)